dashboard/dashboard-faq.md

```diff
@@ -0,0 +1,60 @@
+---
+title: TiDB Dashboard FAQ
+summary: Learn about the frequently asked questions (FAQs) and answers about TiDB Dashboard.
+category: how-to
+---
+
+# TiDB Dashboard FAQ
+
+This document summarizes the frequently asked questions (FAQs) and answers about TiDB Dashboard.
+
+## Access TiDB Dashboard
+
+### When the firewall or reverse proxy is configured, I am redirected to an internal address other than TiDB Dashboard
+
+When multiple PD instances are deployed in a cluster, only one of the PD instances actually runs the TiDB Dashboard service. If you access other PD instances than this Dashboard-running one, your browser redirects you to another address. If the firewall or reverse proxy is not properly configured for accessing TiDB Dashboard, after the visiting the Dashboard, you might be redirected to an internal address that is protected by the firewall or reverse proxy.
+
+- See [TiDB Dashboard Multi-PD Instance Deployment](/dashboard/dashboard-ops-deploy.md#) to learn the working principle of TiDB Dashboard with multiple PD instances.
+- See [Use TiDB Dashboard through Reverse Proxy](/dashboard/dashboard-ops-reverse-proxy.md) to learn how to correctly configure reverse proxy.
+- See [Improve TiDB Dashboard Security](/dashboard/dashboard-ops-security.md) to learn how to correctly configure the firewall.
+
+### When dual network cards are deployed, TiDB Dashboard cannot be accessed using another network card
+
+For security reasons, TiDB Dashboard on PD only monitors the IP addresses specified during deployment (that is, it only listens on one network card), nsot on `0.0.0.0`. Therefore, when multiple network cards are installed on the host, you cannot access TiDB Dashboard using another network card.
```

Comment:
```suggestion
For security reasons, TiDB Dashboard on PD only listens the IP addresses specified during deployment (that is, it only listens on one network interface) instead of `0.0.0.0`. Therefore, when there are multiple network interfaces on the host, you cannot access TiDB Dashboard via another network interface.
```